src/ziggurat/middleware/json.clj

```diff
@@ -0,0 +1,45 @@
+(ns ziggurat.middleware.json
+  "This namespace defines middleware methods for parsing JSON strings.
+   Please see [Ziggurat Middleware](https://github.com/gojek/ziggurat#middleware-in-ziggurat) for more details.
+  "
+  (:require [cheshire.core :refer :all]
```

Comment:
please use namespaced function names instead of `:refer :all` https://github.com/bbatsov/clojure-style-guide#prefer-require-over-use